src/components/attendees/styles.css

```
@@ -0,0 +1,94 @@
+table {
+	border-collapse: collapse;
+	border-spacing: 5px;
+}
+
+tr {
+	text-align: start;
+}
+th {
```

```suggestion

th {
```
I think we should have a new line here for consistency.